llvm/test/CodeGen/RISCV/stack-realignment-with-variable-sized-objects.ll

```diff
@@ -127,3 +135,8 @@ define void @caller(i32 %n) {
   call void @callee(ptr %1, ptr %2)
   ret void
 }
+;; NOTE: These prefixes are unused and the list is autogenerated. Do not add tests below this line:
```

Comment:
Why are we adding unused check-prefixes?

Reply:
Just add some test coverage. Some tests will have an assertion without this patch.
I can just add a new test file if you prefer that way.

Reply:
Why can't we add the RUN line with only the common check-prefix instead of 2 prefixes? We seem to randomly get patches where someone will remove these unused prefixes. For example fab25949688d7b038ee047d4e8b44a23803db3bd 10c86197010e675bfbac7f324b9ff04e43117c5f 37b474a20e8a731bec9db7033e02013c07bd6fa4